_posts/2019-05-06-gis-wisdom.md

```diff
@@ -44,20 +44,20 @@ Learn more about the [ArcGIS versioning process](http://desktop.arcgis.com/en/ar
 
 `networking`
 
-In GIS, we build network datasets for navigation.  These networks are based on connectivity.  We use terms such as edges and junctions, but really we’re just establishing resource groups to which we grant connectivity to other sources.
+In GIS, we build network datasets for navigation. These networks are based on connectivity. We use terms such as *edges* and *junctions*, but really we’re just establishing resource groups to which we grant connectivity to other sources.
 
-Likewise, building a personal network helps us determine who and what we allow ourselves to connect with.  A well-thought-out network not only tells us where we can connect but also where we should not. Taking the time to think about this connectivity ensures that we build a quality social network.
+Likewise, building a personal network helps us determine who and what we allow ourselves to connect with. A well-thought-out network not only tells us where we can connect but also where we should not. Taking the time to think about this connectivity ensures that we build a quality social network.
 
-Learn more about [creating an ArcGIS network datasets](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/creating-a-network-dataset.htm) and [how connectivity works](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/understanding-connectivity.htm).
+Learn more about [creating ArcGIS network datasets](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/creating-a-network-dataset.htm) and [how connectivity works](http://desktop.arcgis.com/en/arcmap/latest/extensions/network-analyst/understanding-connectivity.htm).
 
 ### 3. Extract, Translate, and Load (ETL) 
 {: .text-left}
 
 `being adaptable`
 
-You have probably heard the phrase, 'build once, use many'.  When building data, we say this when we want the output to be adaptable and used for many purposes.  Take road centerlines for example.  This dataset should be designed with many uses in mind (such as routing, mapping, locating, asset management, etc.).  With a simple ‘extract, translate, and load’ (ETL), the data should meet the needs of many users.  
+You may have heard the phrase "build once, use many." When building data, we say this when we want the output to be adaptable and used for many purposes. Take road centerlines for example. This dataset should be designed with many uses in mind (such as routing, mapping, locating, asset management, etc.). With a simple "extract, translate, and load" (ETL), the data should meet the needs of many users.  
```

Comment:
So here I hadn't actually heard the phrase "build once, use many," although it's a great phrase, and I'm sure it's common and maybe I'm the outlier here ;) Regardless, I thought "You may have heard the phrase..." was a bit more open in case people haven't heard it?

Reply:
first batch looks good, thanks!